### PR TITLE
Feat/slack event app

### PR DESCRIPTION
**Pull request by Tinusf**

### CODE REVIEW

File: lambda/slack_response/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# slack_response
+Denne lambdaen skal ta imot en request fra ```slack_command```-lambdaen og lage en response og gi
+den til slack sin response_url.
+
+## Virkemåte
+Denne funker ved at den først henter ut de 10 nærmeste fageventene til knowit på knowit sin
```
Comment (aslettemark) [inline lambda/slack_response/README.md]: neste fagarrangementene

File: lambda/slack_response/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+tilfeldig kode og så sjekke om den blir alt brukt eller ikke. Og så er det bare å bygge
+json-objektet som skal returnes og vises fram som en Slack interactive block app og sende den 
+til ```response_url```-en. 
+
+## Setup
+
+For å sette opp denne lambdaen må du først hente en ```credentials.json``` fil på 
+https://developers.google.com/calendar/quickstart/python, vi har laget en egen google-konto til 
+å gjøre dette. Deretter så kan du gjøre ```get_token.py``` filen for å lage en token i en 
+```token.pickle``` fil. Da kan du kjøre ```deploy_to_aws.sh``` for å deploye lambdaen.
```
Comment (aslettemark) [inline lambda/slack_response/README.md]: utdatert

File: lambda/slack_response/slack_response.py [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+    # This event is when someone typed in /arrangement in slack.
+    if event_type == "/event_button":
+        blocks = create_blocks(calendar_events)
+        send_response(blocks, response_url)
+    # This event is when someone clicked on `få kode` on some event.
+    elif event_type == "click_action":
+        event_id_clicked = event["event_id_clicked"]
+        now = int(datetime.datetime.now().timestamp())
+        to = now + 24 * 60 * 60  # TODO
```
Comment (Tinusf) [inline lambda/slack_response/slack_response.py]: This should be fixed first.

Comment (Tinusf) [reply lambda/slack_response/slack_response.py]: fixed.

**Pull request merged**